src/Command/RunCommand.php

```diff
@@ -128,12 +137,38 @@ protected function configure(): void
                 Container::DEFAULT_CONFIG_FILE
             )
             ->addOption(
-                'coverage',
+                self::OPTION_COVERAGE,
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Path to existing coverage directory',
+                sprintf(
+                    'Path to existing coverage directory (conflicts with --%s and --%s)',
+                    self::OPTION_COVERAGE_XML_DIR,
+                    self::OPTION_JUNIT_LOG_FILE
+                ),
                 Container::DEFAULT_EXISTING_COVERAGE_PATH
             )
+            ->addOption(
+                self::OPTION_COVERAGE_XML_DIR,
+                null,
+                InputOption::VALUE_REQUIRED,
+                sprintf(
+                    'Path to existing xml coverage directory, require --%s (conflicts with --%s)',
+                    self::OPTION_JUNIT_LOG_FILE,
+                    self::OPTION_COVERAGE
+                ),
+                Container::DEFAULT_EXISTING_COVERAGE_XML_PATH
+            )
+            ->addOption(
+                self::OPTION_JUNIT_LOG_FILE,
```

Comment:
I don't mind constants, and I'd probably use them in a similar situation, but this goes out of line with other parts of the file tbh.

But don't take just my word for it. Let's hear what others think.

Reply:
I can refactor all command. From my education and previous experience: in case of using the same value more than one in same context -- it should be variable or constant. This could help with decreasing number of mistakes. But anyway, as usual: your project -- your rules :wink:

Reply:
I agree, but a hodgepodge of constants and string literals is, well, confusing. One starts asking why there's a constant for this, and not a constant for that... In this situation it might be more important to avoid confiscation over following the general (correct) rule for constants. But let's hear out other reviewers.

I don't think it'll be a terrible idea to convert this file to use constants for all commands, but surely in a different, much shorter PR.

Reply:
Good point for second PR :wink: